commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderServiceImpl.java

```diff
@@ -338,6 +340,29 @@ public int getCommerceOrdersCount(long groupId, long commerceAccountId)
 			groupId, commerceAccountId);
 	}
 
+	@Override
+	public List<CommerceOrder> getPendingCommerceOrders(
```

Comment:
@alecsloan This logic introduces a security issue,  the remote services can be invoked via JSONWS or SOAP, that means that a user can pass whatever account id and we are not checking for permission.